src/System.CommandLine.Tests/OptionTests.MultipleArgumentsPerToken.cs

```diff
@@ -86,6 +96,76 @@ public void When_option_is_not_respecified_and_limit_is_reached_then_the_followi
                         .Should()
                         .BeEquivalentTo("some-arg");
                 }
+
+                [Theory]
+                [InlineData("--option 1 --option 2")]
+                [InlineData("xyz --option 1 --option 2")]
+                [InlineData("--option 1 xyz --option 2")]
```

Comment:
Should there be one more option with xyz at the end?
```suggestion
                [InlineData("--option 1 xyz --option 2")]
                [InlineData("--option 1 --option 2 xyz")]
```